### PR TITLE
Always show basal rates on watch, whether temporary or normal.

### PR DESCRIPTION
**Pull request by andrew-warrington**

### CODE REVIEW

File: app/src/main/java/info/nightscout/androidaps/plugins/Wear/wearintegration/WatchUpdaterService.java [hunk 1]
```diff
@@ -636,10 +636,12 @@ private String generateStatusString(Profile profile, String tempBasal, String io
     @NonNull
     private String generateBasalString(TreatmentsInterface treatmentsInterface) {
 
-        String basalStringResult = "-.--U/h";
+        String basalStringResult;
         TemporaryBasal activeTemp = treatmentsInterface.getTempBasalFromHistory(System.currentTimeMillis());
         if (activeTemp != null) {
-            basalStringResult = activeTemp.toStringShort();
+            basalStringResult = activeTemp.toStringVeryShort();
```
[the remark this answers is not in the document]
Comment (andrew-warrington) [reply app/src/main/java/info/nightscout/androidaps/plugins/Wear/wearintegration/WatchUpdaterService.java]: Will do.

File: app/src/main/java/info/nightscout/androidaps/plugins/Wear/wearintegration/WatchUpdaterService.java [hunk 2]
```diff
@@ -636,10 +636,12 @@ private String generateStatusString(Profile profile, String tempBasal, String io
...
+        } else {
+            basalStringResult = DecimalFormatter.to2Decimal(MainApp.getConfigBuilder().getProfile().getBasal()) + "U/h";
```
[the remark this answers is not in the document]
Comment (andrew-warrington) [reply app/src/main/java/info/nightscout/androidaps/plugins/Wear/wearintegration/WatchUpdaterService.java]: To be sure I have understood: You mean to show the basal rate if that boolean is false, and 100% if it is true? Makes sense. (I took my code from the NSClient overview so this must be why I missed the DanaR related stuff.)

**Comment by andrew-warrington** [comment]
@AdrianLxM - both requested changes are made. I tested to the extent I could under NSClient.

**Comment by AdrianLxM** [comment]
Looks good. Tested it for a few days and it shows on the watch what I expect it to show.
Thank you very much @andrew-warrington

**Pull request merged**